ext/nnstreamer/tensor_filter/tensor_filter_snpe.cc

```diff
@@ -300,6 +320,49 @@ snpe_subplugin::configure_instance (const GstTensorFilterProperties *prop)
           } else {
             nns_logw ("Unknown runtime (%s), set CPU as default.", options[op]);
           }
+        } else if (g_ascii_strcasecmp (option[0], "OutputTensor") == 0) {
+          /* the tensor name may contain ':' */
+          gchar *_ot_str = g_strjoinv (":", &option[1]);
+          gchar **names = g_strsplit (_ot_str, ";", -1);
+          guint num_names = g_strv_length (names);
+          outputstrListHandle = Snpe_StringList_Create ();
+          for (guint i = 0; i < num_names; ++i) {
+            if (g_strcmp0 (names[i], "") == 0) {
+              nns_loge ("Given tensor name with index %u is invalid.", i);
```

Comment:
Doesn't need any ```break`` or ```return``` here?

Reply:
Thanks! I make it throws exception on error case. PTAL